### DISCUSSION TITLE
Does LTMEAT Print actually work?

### DISCUSSION THREAD

**Discussion by S-PScripts** [3kh0/ext-remover#810] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I do all the instructions. I do CMD+P on the manifest and duplicate it, then click the switch for allow access to file URLS. Both of the manifest tabs close. Pages still get blocked by the extension and I can still visit the manifest.

**Comment by AshtonDavies** [comment]
LTMEAT Print does not keep the extension frozen very long usually.

**Reply by S-PScripts** [reply]
Oh. But it doesn't even last for 10 seconds though...

**Comment by S-PScripts** [comment, marked as answer]
ffun
